templates/go/scaffolding/instanced-cloudevent/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	ce "github.com/lkingland/func-runtimes/go/cloudevents"
```

Comment:
I would expect this repository to be transferred to the Boson project once this string of PRs comprising the Scaffolding POC are a bit more mature.